jsapp/js/components/formEditors.es6

```diff
@@ -159,10 +159,33 @@ export class ProjectDownloads extends React.Component {
     dataInterface.getAssetExports(this.props.asset.uid).done((data)=>{
       if (data.count > 0) {
         data.results.reverse();
-        this.setState({exports: data.results});
+        let results = data.results.map(result => {
+          if (result.data) {
+            let lang = result.data.lang,
+                langDescription = lang;
+
+            // Backend now changes `_default` to `FormPack.constant.UNTRANSLATED` in `ExportTaskViewSet`.
+            // `FormPack.constant.UNTRANSLATED` constant equals `None`.
+            // Old exports can still have `_default` as value for `lang` though.
+            if (lang === '_default' || lang === null) {
+              langDescription = t('Default');
+            }
+            // Because `xml` can be a code for an existing language (https://en.wikipedia.org/wiki/Malaysian_Sign_Language),
+            // `_xml` is now sent to backend instead of `xml`.
+            // Backend also changes `_xml` to `FormPack.constant.UNSPECIFIED_TRANSLATION`
+            // `FormPack.constant.UNSPECIFIED_TRANSLATION` constant equals `False`.
+            else if (lang === '_xml' || lang === false) {
+              langDescription = t('XML values and headers');
```

Comment:
Admirable, but this wraps onto a second line:
![image](https://user-images.githubusercontent.com/2085013/47331818-54e53980-d64b-11e8-9cb1-63be43588239.png)

Also, for every language it's the `values and headers`, but we probably wouldn't want to say `English (en) values and headers` either.

Reply:
What's the suggestion? Only `XML` in caps?